crmsh/bootstrap.py

```diff
@@ -2152,15 +2152,24 @@ def set_cluster_node_ip():
             break
 
 
+def stop_cluster_related_service(remote_addr=None):
```

Comment:
def stop_cluster_related_service(services, remote_addr=None):
    """
    Disable and stop cluster related service
    """
    for service in services:
        if utils.service_is_available(service, remote_addr=remote_addr):
            status("Stopping the {}".format(service))
            utils.stop_service(service, disable=True, remote_addr=remote_addr)